src/messaging/ExchangeDelegate.h

```diff
@@ -72,6 +76,15 @@ class DLL_EXPORT ExchangeDelegate
      *  @param[in]    ec            A pointer to the ExchangeContext object.
      */
     virtual void OnExchangeClosing(ExchangeContext * ec) {}
+
+    virtual ExchangeTransport * GetTransport(ReliableMessageMgr * rmMgr, SecureSessionMgr * sessionMgr)
+    {
+        mTransport.Init(rmMgr, sessionMgr);
+        return &mTransport;
+    }
```

Comment:
I am not sully understand why app need to get transport info, the concept of exchange is it allows two node establish a virtual connection, like a pipe. we just need to send message at the one end of exchange and the message will be delivered to the other end of the exchange

Reply:
Discussed this in the slack channel. Also the latest commit updates the name to remove confusion.